config/sys.config

```diff
@@ -100,6 +100,8 @@
                       'CN779' => [779.5, 779.7, 779.9],
                       'AU915' => [916.8, 917.0, 917.2, 917.4, 917.5, 917.6, 917.8, 918.0, 918.2],
                       'AS923' => [923.2, 923.4, 923.6, 923.8, 924.0, 924.2, 924.4, 924.5, 924.6, 924.8],
+                      'AS923-2' => [921.5, 921.6, 921.8, 922.0, 922.2, 922.4, 922.6, 922.7, 922.8, 923.0],
```

Comment:
```suggestion
                      'AS923-AS2' => [921.5, 921.6, 921.8, 922.0, 922.2, 922.4, 922.6, 922.7, 922.8, 923.0],
```

Reply:
> We discussed internally (@JayKickliter @michaeldjeffrey) and would like to go towards this naming convention.

My bad. Our agreed convention aligns with what you were doing. Only the 1st one needs the `-1` appended.